.travis.yml

```
@@ -2,31 +2,28 @@ os: linux
 
 dist: xenial
 
-language: c
+language: python #this string allows to test 2 versions of python, unlike language: c
 
-
-matrix:
-    include:
-        - env: PYTHON="python3.6"
-        - env: PYTHON="python2.7"
+python:
+  - "2.7"
+  - "3.6"
 
 sudo: required
 
-
 before_install:
-  - if [[ $PYTHON = "python3.6" ]]; then sudo add-apt-repository -y ppa:jonathonf/python-3.6; sudo apt-get update -y; sudo apt-get install -y python3.6; fi
-  - PYTHON_PATH=$(which $PYTHON)
-  - sudo apt-get install -y python-dev autoconf libtool make unzip g++ pkg-config cmake
-
+  - if [[ $TRAVIS_PYTHON_VERSION = "3.6" ]]; then sudo add-apt-repository -y ppa:jonathonf/python-3.6; sudo apt-get update -y; sudo apt-get install -y libpython3.6-dev; fi
+  - sudo apt-get install -y autoconf libtool make unzip g++ pkg-config cmake
 
 before_script:
   - python --version
+  - sudo ls /usr/lib/x86_64-linux-gnu/libpython*
+  - LIBPYTHONPATH=$(sudo find /usr/lib/x86_64-linux-gnu/libpython"$TRAVIS_PYTHON_VERSION"*.so)
```

something is very wrong here, why not simply
```suggestion
  - LIBPYTHONPATH=$(ls "/usr/lib/x86_64-linux-gnu/libpython$TRAVIS_PYTHON_VERSION*.so")
```
?

Also, `PYTHONPATH` has completely different meaning in Python world, so better be explicit and name smth like `PATH_TO_LIBPYTHON`

---

Of course. I will take into account